src/lib/error/ErrorReport.ts

```diff
@@ -131,16 +135,25 @@ export class ErrorReport extends Error {
   }
 
   public toObject() {
-    return {
+    const r = {
       errorId: this.errorId,
       timestamp: this.timestamp,
       kind: this.kind,
-      message: this.message,
+      message: this.maybeTruncateField(this.message),
       errorDetails: this.errorDetails,
-      stack: this.stack,
+      stack: this.maybeTruncateField(this.stack),
       env: this.env,
       ...(this.originalError.code ? { code: this.originalError.code } : null),
     }
+    console.log(r)
```

Comment:
This can be set to production mode again (removing the `r` variable and the `console.log`)